web-client/src/components/WorkForReviewComponent.jsx

```diff
@@ -0,0 +1,394 @@
+                <div className={style.author}>
+                    <span className={style.shared}>Shared by</span>
+                    <p className={style.leftTitle}>
+                        <img src={props.path} className={style.photo} alt={props.alternativeText}></img> 
+                    </p>
+                    <p className={style.rightTitle}>
+                        <span className={style.authorName}>{props.author}</span>
+                        <span className={style.degree}>{props.degree}</span>
+                        <span className={style.university}>{props.university}</span>
+                    </p>
+                </div>
+                <div className={style.authors}>
+                    <span className={style.other}>Other authors</span>
+                    <span className={style.otherName}>{props.authors}</span>
+                </div>
+                <p className={style.text}>{props.text}</p>
+
+                <Button variant='outlined' color="primary" 
+                    className={style.btn1}>Download full work</Button>
+                <Button variant='contained' color="primary" onClick={handleClickOpen} 
+                    className={style.btn1}>Add review</Button>
+
+                {/* All Dialog in Popup */}    
+                <Dialog 
+                    className={style.mainDialog}
+                    open = {open}
+                    onClose = {handleClose}
+                >
+                    <form
+                        className={style.form}
+                        noValidate
+                        onSubmit={handleSubmit((data) => {
+                        data.stars = value
+                        alert(JSON.stringify(data))
+                        })}
+                    >
+                        <DialogTitle id='alert-dialog-title' className={style.dialogTitle}>
+                            {'Add your review'}
+                        </DialogTitle>
+                        <DialogContent>
+                            <Box component='fieldset' mb={3} borderColor='transparent' className={style.range}>
+                                <Typography component='legend' className={style.grade}>Grade:</Typography>
+                                <Rating className={style.stars}
+                                    max={3}
+                                    value={value}
+                                    onChange={(event, newValue) => {
+                                        if(newValue !== null)
+                                            setValue(newValue);
+                                    }}
+                                    onChangeActive={(event, newHover) => {
+                                        setHover(newHover);
+                                    }}
+                                />
+                                {value !== null && <Box ml={2} className={style.label}>{labels[hover !== -1 ? hover : value]}</Box>}
```

Comment:
How it works?

Reply:
![image](https://user-images.githubusercontent.com/44361068/103699932-f98d9080-4fa3-11eb-88f1-3daa59f33ae6.png)

displays the appropriate value [rejected, correct, accepted]